Makefile

```diff
@@ -1,39 +1,43 @@
 curr_dir := $(shell pwd)
 
 
-DOCKER_NO_CACHE ?=
-CONT_NAME = nvme
-CONT_VERS = latest
-REMOTE_REPO =
+DOCKER_NO_CACHE := --no-cache
+CONT_NAME := nvme
+CONT_VERS := latest
+REMOTE_REPO :=
+SPDK_VERSION := $(shell cd spdk;git -C . describe --tags --abbrev=0 | sed -r 's/-/\./g')
 
 ## init: Initialize the spdk submodule
 .PHONY: init
 init:
 	@echo "dir: $(curr_dir)/spdk"
 	cd $(curr_dir)/spdk && \
-	git submodule update --init --recursive && \
-	git -C . describe --tags --abbrev=0 | sed -r 's/-/\./g' > VERSION
+	git submodule update --init --recursive
 
-## image: Build spdk rpms
+## image: Build spdk image and rpms
 .PHONY: spdk
 spdk:
 	tar -czf docker/spdk.tar.gz -C . spdk
```

Comment:
Why do you tar and copy the spdk into the container to then untar again. Why not just copy the folder?
Or alternatively check out the exact commit of the submodule in the container.